### PR TITLE
Fix UMD

### PR DESCRIPTION
**Pull request by IceOnFire**

### CODE REVIEW

File: papaparse.js [hunk 1]
```diff
@@ -3,7 +3,26 @@
...
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory();
+  }
+	else
+	{
+    // Browser globals (root is window)
+    root.returnExports = factory();
+  }
+}(this, function()
```
Comment (mholt) [inline papaparse.js]: Can you please fix the indentation here? It's rather inconsistent.

**Comment by mholt** [comment]
Sorry, but what does this do exactly? What's broken?

**Comment by IceOnFire** [comment]
This pull request fixes issue https://github.com/mholt/PapaParse/issues/297, which in short is: JSPM fails to load papaparse as an AMD module. I switched my project to my own fork (which contains this fix) and it works again.

The UMD definition above is a bit more standard, since it refers to [this format](https://github.com/umdjs/umd).

**Comment by mholt** [comment]
Alrighty - thanks!

**Comment by mholt** [comment]
@IceOnFire I've been getting reports that "global" is not defined -- could you do me a favor and fix that? :) It was working before this PR. https://twitter.com/xirzon/status/722234398406418433

**Comment by IceOnFire** [comment]
I'm working on it, sorry for the trouble.

**Pull request merged**